cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,24 @@
+# This uninstall script is copied from the CMake wiki page:
+# https://cmake.org/Wiki/CMake_FAQ#Can_I_do_.22make_uninstall.22_with_CMake.3F
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
```

Comment:
I am open to keeping the delimiter as newlines if this significantly simplifies the implementation. Like @psigen said, it's pretty uncommon to encounter newline characters in filenames in the wild - and you likely have bigger problems than Chimera if you do.

<img src="https://mlkshk-ada.kxcdn.com/r/ZWGV"/>